lib/pbench/server/database/alembic/env.py

```diff
@@ -28,11 +28,11 @@
 handler = logging.handlers.SysLogHandler("/dev/log")
 log.addHandler(handler)
 
-# add your model's MetaData object here
-# for 'autogenerate' support
+# add your model's MetaData object here for 'autogenerate' support:
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-target_metadata = [Database.Base.metadata]
+
+target_metadata = Database.Base.metadata
```

Comment:
I don't understand this comment in light of this change:  if we can use multiple models (no apostrophe required...), then presumably `target_metadata` would be assigned to a list (so the previous code was not wrong).  If we're already adding our model here (which we are) then I'm not seeing how the comments in lines 32-33 help.

Since we're only using a single model, I have no problem with specifying an atom instead of a single element list.  And, I think line 31 is constructive.  But, I don't think that lines 32-33 are helpful, now.